crates/hir_ty/src/tests/traits.rs

```diff
@@ -174,24 +174,41 @@ fn test() {
 //- /core.rs crate:core
 #[prelude_import] use ops::*;
 mod ops {
-    trait Try {
-        type Output;
-        type Residual;
+    mod try_trait {
+        pub trait Try: FromResidual {
+            type Output;
+            type Residual;
+        }
+        pub trait FromResidual<R = <Self as Try>::Residual> {}
     }
+
+    pub use self::try_trait::FromResidual;
+    pub use self::try_trait::Try;
+}
+
+#[prelude_import] use convert::*;
```

Comment:
multiple prelude imports don't work like that AFAIK, I think this is just working because you only need the `result` one anyway?

Reply:
ups. i'll change that :)